tests/integration/bootstrap/FeatureContext.php

```diff
@@ -80,4 +80,18 @@ public function disableDataImport()
 		$config = \Kohana::$config->load('features');
 		$config->set('data-import.enabled', false);
 	}
+
+	/** @BeforeScenario @cdnEnabled */
+	public function enableCdn()
```

Comment:
You should remove these before this lands, since they didn't work anyway.
Given we don't have a convenient way to enable/disable the CDN during test runs, we should possibly just run with the CDN enabled for all tests since thats the production state. @willdoran what do you think?